### ISSUE TITLE
Move transform on top of inspector and always accessible

### ISSUE THREAD

**Issue by TrickyFatCat** [Redot-Engine/redot-proposals#32]
### Describe the project you are working on

-

### Describe the problem or limitation you are having in your project

Transform is the most frequently used parameters during game development. In Redot (Godot) inspector it situated in the bottom part of it and hidden by default. In some cases it's quite annoying to find it and change by hand. Unlike many other game engines
<img width="1280" alt="Transfrom" src="https://github.com/user-attachments/assets/0038c58e-2ef2-4b83-96b1-30c91baa98de">

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be great to see transform placed always on top and open to edit by default.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

It's a common practice for many game engines to place transform on top of their inspector/details, because it's the most common property to change.

**Comment by Spartan322** [comment]
I'm not sure it should be moved by default, but being open by default perhaps. Perhaps if fields in the inspector could be favorited instead?

**Comment by TrickyFatCat** [comment]
@Spartan322 well, it's the most common thing between many game engines. Transform is the most used properties, especially in 3D games. Put it somewhere in the middle/bottom part of the inspector makes it quite hard to use, especially, when you work on levels and UI.

**Comment by Spartan322** [comment]
Sure, the issue is more that the inspector is designed in a way that inheritance is displayed in an ordered fashion, by default I don't really agree it should be set at the top by default because that ruins the organization of the inspector, least without user opt-in, say with favorited variables in the inspector.

**Comment by TrickyFatCat** [comment]
@Spartan322 it's an industrial standard for a rally good reason, which already described.  I believe the majority of people do want a good UX out of the box for a basic game development. Favourites is a good idea, but it's for power users, rather than the majority. I don't get this "inspector organization" argument at all, as it's not good in the first place, because of this issue.

**Comment by tumblewed** [comment]
Adding the ability to favorite or pin categories to the top of the inspector would be a great fix without changing the inheritance structure. The engine can pin the Transform field by default for all Node3Ds, which will create a duplicate of the field so both workflows remain intact.

**Comment by ArseniyMirniy** [comment]
I agree that it's the most basic feature relevant to EVERY SINGLE NODE and should never be on the bottom of the list. Each time I need it, it takes about 1–2 minutes to locate it in the inspector to place the item in a specific spot of the scene. Same, when I need to animate them.
